src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -923,6 +948,11 @@ public sealed class ValidateRangeAttribute : ValidateEnumeratedArgumentsAttribut
         /// </summary>
         private Type _promotedType;
 
+        ValidateRangeKind? _rangeKind;
+
+        private readonly static Type _zeroType = 0.GetType();
+        private readonly static string _zeroTypeName = 0.GetType().Name;
```

Comment:
Please use s_zeroType for static constant.
And maybe `_zeroTypeName = s_zeroType.Name; `